.github/actions/spelling/patterns.txt

```diff
@@ -17,3 +17,5 @@ data:[a-zA-Z=;,/0-9+-]+
 El proyecto .* diferentes
 # Package family names
 \b[-.A-Za-z0-9]+_[a-z0-9]{13}\b
+# Locales for name normalization
+\b\p{Lu}{2,3}(-(CANS|CYRL|LATN|MONG))?-\p{Lu}{2}(?![A-Z])(?:-VALENCIA)?\b
```

Comment:
I'd generally suggest doing:
```suggestion
\b\p{Lu}{2,3}(?:-(?:CANS|CYRL|LATN|MONG))?-\p{Lu}{2}(?![A-Z])(?:-VALENCIA)?\b
```
Basically, if you aren't capturing you should tell perl you aren't.

Otherwise, if you have a rule like:
```
\b([A-Za-z])\1{3,}\b
```
(which is earlier in this file), you may be unhappy.

Reply:
The entire file is run as one big regex?  I would assume that the capture reference would be per-line.  I don't mind changing it, I just didn't expect that the captures really mattered for this use case.

Reply:
Yeah, it's run as one big regex.

It's a lazy optimization that only trips things up occasionally.

I need to look into whether it's worth treating each line as a unique regex / how to make that work.